docs/standard/choosing-core-framework-server.md

```diff
@@ -57,15 +57,17 @@ Performance and scalability are especially relevant for microservices architectu
 
 To install applications with dependencies on different versions of .NET, we recommend .NET Core. .NET Core supports side-by-side installation of different versions of the .NET Core runtime on the same machine. This side-by-side installation allows multiple services on the same server, each of them on its own version of .NET Core. It also lowers risks and saves money in application upgrades and IT operations.
 
-Side-by-side installation isn’t possible with .NET Framework. It’s a Windows component, and only one version can exist on a machine at a time. Each version of .NET Framework replaces the previous version. If you install a new app that targets a later version of .NET Framework, you might break existing apps that run on the machine, because the previous version was replaced.
+Side-by-side installation isn't possible with .NET Framework. It's a Windows component, and only one version can exist on a machine at a time. Each version of .NET Framework replaces the previous version. If you install a new app that targets a later version of .NET Framework, you might break existing apps that run on the machine, because the previous version was replaced.
 
 ## When to choose .NET Framework
 
 .NET Core offers significant benefits for new applications and application patterns. However, .NET Framework continues to be the natural choice for many existing scenarios, and as such, .NET Framework isn't replaced by .NET Core for all server applications.
 
+[!INCLUDE [dotnet-behavioral-differences](../../includes/dotnet-behavioral-differences.md)]
```

Comment:
I read through the whole issue #762, and I actually think it can/should be closed without any action on our part. The action taken was for the engineering team to start labeling their PRs with breaking-change. We already have docs (that were probably written after that issue was created in 2016) that detail unsupported technologies: https://docs.microsoft.com/en-us/dotnet/core/porting/net-framework-tech-unavailable. And a whole TOC on breaking changes: https://docs.microsoft.com/en-us/dotnet/core/compatibility/.